examples/example_03_base_workflow.py

```diff
@@ -0,0 +1,88 @@
+# -*- coding: utf-8 -*-
+# pylint: disable=invalid-name
+"""Run simple DFT calculation"""
+
+from __future__ import print_function
+from __future__ import absolute_import
+
+import sys
+import click
+
+from aiida.engine import run, run_get_node
+from aiida.orm import Code, Dict, StructureData
+from aiida.common import NotExistent
+from aiida.plugins import WorkflowFactory
+
+import pymatgen as mg
+
+GaussianBaseWorkChain = WorkflowFactory('gaussian.base')
+
+
+def example_dft(gaussian_code):
```

Comment:
Does this example produce the CalDSu error and is saved by the handler? If so, maybe good to write a small comment on it.

Reply:
I removed the example from this PR, as it doesn't reproduce the CalDSu problem. I will re-introduce it back when dealing with the ASyTop error (#14)